### PR TITLE
[Admin] Show order details

### PR DESCRIPTION
**Pull request by ghost**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`

`explain select * from `orders` where `orders`.`id` = '5' limit 1\G
`
> *************************** 1. row ***************************
>            id: 1
>   select_type: SIMPLE
>         table: orders
>    partitions: NULL
>          type: const
> possible_keys: PRIMARY
>           key: PRIMARY
>       key_len: 4
>           ref: const
>          rows: 1
>      filtered: 100.00
>         Extra: NULL
> 1 row in set, 1 warning (0.00 sec)

`explain select count(*) as aggregate from `order_details` where `order_details`.`order_id` = '5' and `order_details`.`order_id` is not null\G`

> *************************** 1. row ***************************
>            id: 1
>   select_type: SIMPLE
>         table: order_details
>    partitions: NULL
>          type: ref
> possible_keys: order_details_order_id_foreign
>           key: order_details_order_id_foreign
>       key_len: 4
>           ref: const
>          rows: 3
>      filtered: 100.00
>         Extra: Using where; Using index
> 1 row in set, 1 warning (0.00 sec)

`explain select * from `order_details` where `order_details`.`order_id` = '5' and `order_details`.`order_id` is not null limit 5 offset 0\G`

> *************************** 1. row ***************************
>            id: 1
>   select_type: SIMPLE
>         table: order_details
>    partitions: NULL
>          type: ref
> possible_keys: order_details_order_id_foreign
>           key: order_details_order_id_foreign
>       key_len: 4
>           ref: const
>          rows: 3
>      filtered: 100.00
>         Extra: Using index condition
> 1 row in set, 1 warning (0.00 sec)

`explain select * from `products` where `products`.`id` in ('2', '4', '18')\G`

> *************************** 1. row ***************************
>            id: 1
>   select_type: SIMPLE
>         table: products
>    partitions: NULL
>          type: range
> possible_keys: PRIMARY
>           key: PRIMARY
>       key_len: 4
>           ref: NULL
>          rows: 3
>      filtered: 100.00
>         Extra: Using where
> 1 row in set, 1 warning (0.00 sec)

`explain select * from `images` where `images`.`product_id` in ('2', '4', '18')\G`

> *************************** 1. row ***************************
>            id: 1
>   select_type: SIMPLE
>         table: images
>    partitions: NULL
>          type: ALL
> possible_keys: images_product_id_foreign
>           key: NULL
>       key_len: NULL
>           ref: NULL
>          rows: 30
>      filtered: 20.00
>         Extra: Using where
> 1 row in set, 1 warning (0.00 sec)

### CODE REVIEW

File: resources/views/admin/pages/orders/show.blade.php [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+                    <th class="col-md-1">{{ __('orders.admin.show.quantity_col') }}</th>
+                    <th class="col-md-2">{{ __('post.admin.list.action_col') }}</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  @foreach ($orders as $order )
+                  <tr>
+                    <td>
+                        @if (count($order['product']['images']))
+                          <img style="width:100px;height:100px" src="{{ $order['product']['images'][0]->image_url }}" alt="">
```
Comment (at-xinhnguyen) [inline resources/views/admin/pages/orders/show.blade.php]: bỏ vào `css file` đi em, sau này có muốn chỉnh sửa kích thước dũng dễ tim

File: app/Http/Controllers/Admin/OrderController.php [hunk 2]
```diff
@@ -28,4 +28,22 @@ public function index(Request $request)
...
+    /**
+     * Display the specified resource.
+     *
+     * @param int $id order id
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function show($id)
+    {
+        $perPage = config('define.post.limit_rows');
```
[the remark this answers is not in the document]
Comment (at-phuongvu) [reply app/Http/Controllers/Admin/OrderController.php]: dạ cái trang `index` hôm qua á anh. Em chờ merge xong r em lấy cái hqua định nghĩa để dùng luôn cho đỡ conflict. Xong #53 em fix cái này cho đỡ update branch nhiều lần ạ :D

File: resources/views/admin/pages/orders/show.blade.php (hunk 1 above)
Comment (at-phuongvu) [reply resources/views/admin/pages/orders/show.blade.php]: dạ chỗ này em muốn để ở đây để nó ưu tiên lấy cái này cho tất cả các row ảnh nó ko bị chèn khác nhau ạ.

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**